<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Ederson Moura &lt;ederson.moura@resource.com.br&gt;</author>
 <class>BillValidatorFit</class>
 <widget class="QWidget" name="BillValidatorFit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BillValidator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/res/images/app-icon.png</normaloff>:/res/images/app-icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
    border: 2px solid #333333;
    border-radius: 3px;
    padding: 1px 23px 1px 3px;
}

QPushButton:hover, QToolButton:hover {
    background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #ffd9aa, stop : 0.5 #ffbb6e, stop : 0.55 #feae42, stop : 1.0 #fedb74);
}

QPushButton {
    border: 1px solid #333333;
    border-radius: 3px;
    padding: 1px 23px 1px 3px;
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
    color: white;
    font: bold large &quot;Arial&quot;;
    height: 30px;
}

QPushButton:pressed {
    background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #c0dbff, stop : 0.5 #cfd26f, stop : 0.55 #c7df6f, stop : 1.0 #f5f9ff);
    padding-top: 2px;
    padding-left: 3px;
}

QPushButton:on {
    background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #5AA72D, stop : 0.5 #B3E296, stop : 0.55 #B3E296, stop : 1.0 #f5f9ff);
    padding-top: 2px;
    padding-left: 3px;
}

QPushButton:disabled {
    background: transparent #aeaeae;
    padding-top: 2px;        
    padding-left: 3px;
    color: black;
}

QToolButton {
    border: 1px solid #333333;
    border-radius: 3px;
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
}

QProgressBar {
    border: 1px solid black;
    text-align: center;
    padding: 1px;
    border-bottom-right-radius: 7px;
    border-bottom-left-radius: 7px;
    background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 #fff, stop: 0.4999 #eee, stop: 0.5 #ddd, stop: 1 #eee );
    width: 15px;
}

QProgressBar::chunk {
    background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,stop: 0 #78d,stop: 0.4999 #46a,stop: 0.5 #45a,stop: 1 #238 );
    border-bottom-right-radius: 7px;
    border-bottom-left-radius: 7px;
    border: 1px solid black;
}

QLCDNumber {
    background-color:black;
    color:green;
    font: bold large &quot;Arial&quot;;
    height: 30px;
}

QTabWidget, QStackedWidget {
    background: none;
    color: white;
    font: bold large &quot;Arial&quot;;
    height: 30px;
}

QTabWidget::pane {
    border: 0px;
    top: 0.1em;
    border-width: 1px;
    border-style: solid;
    border-color: grey;
}

QTabBar::tab {
    width: 150px;
    margin-left: 0.5;
    margin-right: 0.5;
    border: 1px solid #7a7a7a;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    padding: 15px 19px 15px 19px;
}

QTabBar::tab:!selected {
   margin-top: 2px;
}

QTabBar::tab:selected {
    margin-top: -1px;
   background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
    color: white;
}

QTabBar::tab:hover {
    text-decoration: underline;
     background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #ffd9aa, stop : 0.5 #ffbb6e, stop : 0.55 #feae42, stop :   1.0 #fedb74);
}

QTabBar::tab::bottom {
    border-bottom-left-radius: 4px;
    border-bottom-right-radius: 4px;
    border-top-left-radius: 0px;
    border-top-right-radius: 0px;
    padding: 15px 19px 15px 19px;
}

QComboBox {
	border: 1px solid #333333;
	border-radius: 3px;
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
	padding: 1px 23px 1px 3px;
	min-width: 6em;
    color: white;
    font: bold large &quot;Arial&quot;;
    height: 30px;
}

QComboBox:hover {
    text-decoration: underline;
     background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #ffd9aa, stop : 0.5 #ffbb6e, stop : 0.55 #feae42, stop :   1.0 #fedb74);
}

QComboBox::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 20px;
 
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
     image: url(:res/images/combo-arrow-down.png);
}
 
QComboBox QAbstractView{
	background-color: #4f4f4f;
	color: #999999;
	selection-background-color: #999999;
	selection-color: #4f4f4f;
}

QComboBox QAbstractItemView {
    background-color: #4f4f4f;
    color: white;
   selection-background-color: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #ffd9aa, stop : 0.5 #ffbb6e, stop : 0.55 #feae42, stop : 1.0 #fedb74);
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="selectBox">
     <property name="title">
      <string>Connection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QComboBox" name="comboBoxSerial">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>COM3</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/res/images/serialport.png</normaloff>:/res/images/serialport.png</iconset>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="toolButtonBillValidator">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../images.qrc">
            <normaloff>:/res/images/settings.png</normaloff>:/res/images/settings.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonDisconnect">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Disconnect</string>
          </property>
          <property name="icon">
           <iconset resource="../images.qrc">
            <normaloff>:/res/images/disconnect.png</normaloff>:/res/images/disconnect.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButtonConnect">
          <property name="text">
           <string>&amp;Connect</string>
          </property>
          <property name="icon">
           <iconset resource="../images.qrc">
            <normaloff>:/res/images/connect.png</normaloff>:/res/images/connect.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="selectBox_2">
     <property name="title">
      <string>Status</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Connected:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelConnect">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>YES</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Model:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelFirmware">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>603</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Version:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelVersion">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>3.4.5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Cassette:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelCassetteLevel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>GOOD</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="selectBox_3">
     <property name="title">
      <string>Time Connection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="styleSheet">
         <string notr="true">QLCDNumber { 
color:green;
background-color:black;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Money</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Control Commands:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="5">
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../images.qrc">:/res/images/dollars-bag-300x300.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLCDNumber" name="lcdNumberNoteLoaded">
           <property name="styleSheet">
            <string notr="true">QLCDNumber { 
color:green;
background-color:transparent;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButtonReset">
           <property name="text">
            <string>Reset</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/res/images/devicetest.png</normaloff>:/res/images/devicetest.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButtonBillTables">
           <property name="text">
            <string>Bill Tables</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/res/images/devicetest.png</normaloff>:/res/images/devicetest.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButtonVersion">
           <property name="text">
            <string>Firmware Version</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/res/images/devicetest.png</normaloff>:/res/images/devicetest.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Control Money:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButtonStartPoll">
             <property name="text">
              <string>Poll</string>
             </property>
             <property name="icon">
              <iconset resource="../images.qrc">
               <normaloff>:/res/images/process-start-128x128.png</normaloff>:/res/images/process-start-128x128.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButtonStack">
             <property name="text">
              <string>Stack</string>
             </property>
             <property name="icon">
              <iconset resource="../images.qrc">
               <normaloff>:/res/images/process-stop-128x128.png</normaloff>:/res/images/process-stop-128x128.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pushButtonReturn">
             <property name="text">
              <string>Return</string>
             </property>
             <property name="icon">
              <iconset resource="../images.qrc">
               <normaloff>:/res/images/process-stop-128x128.png</normaloff>:/res/images/process-stop-128x128.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayoutMoveCards"/>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>145</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
